### ISSUE TITLE
“Copy File Path” doesn't work correctly for Media Sources

### ISSUE THREAD

**Issue by SnowCreative** [modxcms/revolution#15326]
### Summary
Right-clicking and selecting “Copy File Path” for a file while viewing in a media source other than “Filesystem” produces an unusable file path.

### Observed behavior
The copied file path is only partial; it does not include any enclosing folders. So, for example, when clients who use this method to grab the URL for a PDF on their site through a media source called "documents", linked to a "documents" folder on the server, they end up with "filename.pdf" instead of "documents/filename.pdf".

### Expected behavior
This action should prepend the "basePath" from the Media Source settings, so that the resulting copied path should look exactly like it would be when using a TV to fetch a file name. In other words, users should end up with a path that, when added to their website's domain name, forms a complete link to that file. 

### Environment
All versions of MODX

**Comment by JoshuaLuckers** [comment]
I propose to have two options instead of changing the current behavior:
- Copy the file path from the media source basePath (the current behavior)
- Copy the file path from the "root" path (your proposal)

Note: at the bottom of the media browser the full path is visible and you can click the icon to copy it.

**Comment by SnowCreative** [comment]
> Note: at the bottom of the media browser the full path is visible and you can click the icon to copy it.

That's true, and a good thing! But, if people use the Files tab and try to copy the path from media sources there, then there is no complete file path to copy when they click on a file in the tree. Having both options you list available in the popup menu would be ideal.

**Comment by Ruslan-Aleev** [comment]
> Copy the file path from the media source basePath (the current behavior)

Not sure about this option. If we use TV with an attached media source, then the path of the source will already be taken into the path.
And if we need a path relative to the media source, then how can we get the path to the media source itself (basePath)?
It just seems to me that these menu options are more needed for clients, but I do not see how clients may need a path relative to the media source.
